includes/admin/class-wc-admin-webhooks-table-list.php

```diff
@@ -207,6 +207,22 @@ protected function get_bulk_actions() {
 		);
 	}
 
+	/**
+	 * Process bulk actions.
+	 */
+	public function process_bulk_action() {
+		$action   = $this->current_action();
+		$webhooks = isset( $_REQUEST['webhook'] ) ? array_map( 'absint', (array) $_REQUEST['webhook'] ) : array(); // WPCS: input var okay, CSRF ok.
+
+		if ( ! current_user_can( 'manage_woocommerce' ) ) {
+			wp_die( esc_html__( 'You do not have permission to edit Webhooks', 'woocommerce' ) );
+		}
+
+		if ( 'delete' === $action ) {
+			WC_Admin_Webhooks::bulk_delete( $webhooks );
```

Comment:
Calling `WC_Admin_Webhooks::bulk_delete()` statically will generate a PHP notice (or warning, I'm not sure which) as this is a non-static method. You need to either declare it as static and then update the other places where it is called or come up with another solution. I personally like to avoid static methods whenever possible, so maybe a better, but more involved solution, is to move all the action methods to `class-wc-admin-webhooks-table-list.php`? Just a suggestion, I'm haven't look into the implications of this change.

Reply:
Ohh yeah, better move into the list table class, since it's already a private method.

Reply:
@claudiosanches The `bulk_delete` method is still required for webhook delete action.